addons/website_crm_partner_assign/controllers/main.py

```diff
@@ -231,35 +229,24 @@ def sitemap_partners(env, rule, qs):
                 yield {'loc': loc}
 
     def _get_partners_values(self, country=None, grade=None, page=0, references_per_page=20, **post):
```

Comment:
We usually don't redefine the arguments in the override
```suggestion
    def _get_partners_values(self, country=None, **post):
```